src/components/items-expander.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+function ItemsExpander({ className, expandLinkClassName, items, renderItem, initialItemCount }) {
+  const { t } = useTranslation('itemsExpander');
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  const showExpander = items.length > initialItemCount + 1;
+
+  return (
+    <div className={className}>
+      {items.filter((_tag, index) => isExpanded || (showExpander && index < initialItemCount)).map(renderItem)}
+      {showExpander && isExpanded && (
+        <a className={expandLinkClassName} onClick={() => setIsExpanded(false)}>{t('collapse')}</a>
+      )}
+      {showExpander && !isExpanded && (
+        <a className={expandLinkClassName} onClick={() => setIsExpanded(true)}>{t('expand', { count: items.length - 10 })}</a>
```

Comment:
Here `10` could be replaced with `initialItemCount`, if I get the purpose of this variable right.

Reply:
Thanks for spotting that